libraries/pbrlib/genglsl/lib/mx_defines.glsl

```diff
@@ -2,3 +2,4 @@
 #define M_PI_INV 1.0/3.1415926535897932384626433832795
 #define M_GOLDEN_RATIO 1.6180339887498948482045868343656
 #define M_FLOAT_EPS 1e-8
+#define M_MAX_RADIANCE_SAMPLES 256
```

Comment:
This value is settable in LightingState so this will override the maximum from there which has a max of 1024.
Maybe it's better to update LightingState to have a maximum and you can set this to a smaller value there and clamp.

Reply:
LightingState is local but now UI Widget set the value on the context.